opensrp-chw/src/main/java/org/smartregister/chw/util/Common.java

```diff
@@ -0,0 +1,36 @@
+package org.smartregister.chw.util;
```

Comment:
I would suggest we move this to the malaria module since it only specific to that module also make it descriptive or add the functions to the utility class

Reply:
But the functionality is not only common to the malaria module, its basically through out the PR